### ISSUE TITLE
[Feature] Is there any plan for fluss to be compatible with kafka protocol?

### ISSUE THREAD

**Issue by wking1986** [alibaba/fluss#89]
### Search before asking

- [X] I searched in the [issues](https://github.com/alibaba/fluss/issues) and found nothing similar.

### Motivation

Kafka is widely used in the field of streaming storage. Being compatible with the Kafka protocol can facilitate the migration of existing systems and reduce the cost of connecting new systems.

### Willingness to contribute

- [ ] I'm willing to submit a PR!

**Comment by polyzos** [comment]
Yes, kafka compatibility is on the [roadmap](https://alibaba.github.io/fluss-docs/roadmap/)
